src/types.ts

```diff
@@ -16,11 +16,13 @@ export interface ObsidianGitSettings {
     autoBackupAfterFileChange: boolean;
     treeStructure: boolean;
 
-    /* Obsolete settings */
-    mergeOnPull?: boolean;  // Migrated to `syncMethod = 'merge'`
+    /**
+     * @deprecated Migrated to `syncMethod = 'merge'`
```

Comment:
I think using `@deprecated` is even better to support highlighting its usage in the editor.

Reply:
I had no idea that even existed, but yeah, that's clearly better.